modules/op-resolver-community-tasks-ref.adoc

```diff
@@ -0,0 +1,411 @@
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: helm-upgrade-from-repo
+spec:
+  tasks:
+    - name: helm-upgrade-from-repo
+      params:
+        - name: helm_repo
+          value: example_helm_repository
+        - name: chart_name
+          value: example_chart_name
+        - name: release_version
+          value: v1.0.0
+        - name: release_name
+          value: helm-release
+        - name: release_namespace
+          value: ''
+        - name: overwrite_values
+          value: ''
+        - name: helm_image
+          value: 'docker.io/lachlanevenson/k8s-helm@sha256:5c792f29950b388de24e7448d378881f68b3df73a7b30769a6aa861061fd08ae'
+      taskRef:
+        kind: Task
+        name: helm-upgrade-from-repo
+----
+
+.Supported parameters for the `helm-upgrade-from-repo` community task
+[cols="1,2,1",options="header"]
+|===
+| Parameter | Description | Default value
+|`helm_repo` |Specify a Helm repository. |
+|`chart_name` |Specify chart name to be deployed. |
+|`release_version` |The Helm release version in semantic versioning format. |`v1.0.0`
+|`release_name` |The Helm release name. |`helm-release`
+|`release_namespace` |The Helm release namespace. |`""`
+|`overwrite_values` |The configuration parameters to overwrite, comma separated. For example: `autoscaling.enabled=true,replicas=1`|`""`
+|`helm_image` |The Helm image to be used. |`docker.io/lachlanevenson/k8s-helm@sha256:5c792f29950b388de24e7448d378881f68b3df73a7b30769a6aa861061fd08ae`
+|===
+
+[discrete]
+[id="op-ctaskref-helm-upgrade-from-source_{context}"]
+== helm-upgrade-from-source
+
+The `helm-upgrade-from-source` community task installs and upgrades a Helm chart in your {OCP} cluster by using Helm.
+
+.Example usage of the `helm-upgrade-from-source` community task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: helm-upgrade-from-source
+spec:
+  tasks:
+    - name: helm-upgrade-from-source
+      params:
+        - name: charts_dir
+          value: example_directory_path
+        - name: release_version
+          value: v1.0.0
+        - name: release_name
+          value: helm-release
+        - name: release_namespace
+          value: ''
+        - name: overwrite_values
+          value: ''
+        - name: values_file
+          value: values.yaml
+        - name: helm_image
+          value: 'docker.io/lachlanevenson/k8s-helm@sha256:5c792f29950b388de24e7448d378881f68b3df73a7b30769a6aa861061fd08ae'
+        - name: upgrade_extra_params
+          value: ''
+      taskRef:
+        kind: Task
+        name: helm-upgrade-from-source
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+  #...
+----
+
+.Supported parameters for the `helm-upgrade-from-source` community task
+[cols="1,2,1",options="header"]
+|===
+| Parameter | Description | Default value
+|`charts_dir` |The directory in the source workspace that contains the Helm chart. |
```

Comment:
Remove "The" here and in the other parameters, I think we don't normally use an article at the start of a parameter description in the table (compare to how the non-community tasks are done)